modules/collect/src/main/java/com/opengamma/strata/collect/timeseries/DenseLocalDateDoubleTimeSeries.java

```diff
@@ -57,7 +62,7 @@
    * array for weekends and providing the different date
    * calculations for each case.
    */
-  static enum DenseTimeSeriesCalculation {
+  enum DenseTimeSeriesCalculation {
```

Comment:
It was clearer with the static (the code reader should not need to know offhand that nested enums are static by default)